modules/nw-ingress-gateway-api-troubleshooting-degraded.adoc

```diff
@@ -0,0 +1,59 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-troubleshooting-degraded_{context}"]
+= Ingress Operator is Degraded due to Gateway API and OSSM conflict
+
+In [product-title] 4.20 and later, if you create a `GatewayClass` resource while a conflicting OpenShift Service Mesh (OSSM) v2.x subscription exists, the `ingress` Cluster Operator (CIO) reports a `Degraded` status. This procedure details how to verify and resolve this conflict.
+
+This occurs because the Gateway API implementation requires OSSM v3.x, which cannot coexist with OSSM v2.x. The CIO detects this conflict, stops the Gateway API provisioning, and reports the `Degraded` status to alert administrators.
```

Comment:
"This occurs" is a vague referent only because a "this clause exists in the previous paragraph.

> The conflict occurs because...